src/UserConfig.elm

```diff
@@ -1651,6 +1651,411 @@ Podemos cerrar su cuenta de inmediato si pensamos que ha:
+    , dontBeAJerk = "อย่าทำตัวเป็นคนไม่ดี"
+    , edit = "แก้ไข"
+    , editEvent = "แก้ไขกิจกรรม"
+    , ended = "จบแล้ว "
+    , endsIn = "จบในอีก "
+    , enterYourEmailAddress = "ใส่ที่อยู่อีเมลของคุณ"
+    , enterYourEmailFirst = "ใส่อีเมลของคุณก่อน"
+    , eventCantBeMoreThan = "กิจกรรมไม่สามารถมากกว่า "
+    , eventCanTStartInThePast = "กิจกรรมไม่สามารถเริ่มในอดีต"
+    , eventDescriptionOptional = "คำอธิบายกิจกรรม (เลือกได้)"
+    , eventDurationText =
+        \isPastEvent durationText eventTypeText ->
+            if isPastEvent then
+                "• นี่เป็นกิจกรรม " ++ eventTypeText ++ " ที่มีระยะเวลา " ++ durationText
+
+            else
+                "• นี่คือกิจกรรม " ++ eventTypeText ++ " ที่จะมีระยะเวลา " ++ durationText
+    , eventName = "ชื่อกิจกรรม"
+    , eventOverlapsOtherEvents = "กิจกรรมนี้ซ้อนทับกับกิจกรรมอื่นๆ"
+    , eventOverlapsWithAnotherEvent = "กิจกรรมนี้ซ้อนทับกับกิจกรรมอื่น"
+    , eventsCanTStartInThePast = "กิจกรรมไม่สามารถเริ่มในอดีต"
+    , failedToJoinEventThereArenTAnySpotsLeft = "ไม่สามารถเข้าร่วมกิจกรรม ไม่มีที่ว่างอีกแล้ว"
+    , failedToJoinThisEventDoesnTExistTryRefreshingThePage = "ไม่สามารถเข้าร่วม กิจกรรมนี้ไม่มีอยู่ (ลองรีเฟรชหน้าเว็บ?)"
+    , failedToLeaveEvent = "ไม่สามารถออกจากกิจกรรม"
+    , faq = "คำถามที่พบบ่อย"
+    , faq1 = "ฉันไม่ชอบที่ meetup.com เรียกเก็บเงิน ส่งอีเมลซับซ้อนถึงฉัน และดูอิ่มตัว ฉันยังอยากลองทำบางสิ่งที่มีความหมายมากขึ้นโดยใช้ "
+    , faq2 = "เพื่อดูว่าเป็นไปได้ในการใช้ที่ทำงานหรือไม่"
+    , faq3 = "ฉันเพียงแค่ใช้เงินส่วนตัวเพื่อโฮสต์มัน สิ่งนี้ไม่เป็นไรเพราะได้ออกแบบมาเพื่อให้มีค่าใช้จ่ายในการทำงานน้อยที่สุด ในกรณีที่ Meetdown ได้รับความนิยมอย่างมากและค่าใช้จ่ายในการโฮสต์เริ่มต้นมากขึ้น ฉันจะขอบริจาค"
+    , faqQuestion1 = "ใครคือผู้ที่อยู่เบื้องหลังทั้งหมดนี้?"
+    , faqQuestion2 = "ทำไมเว็บไซต์นี้ถึงถูกสร้างขึ้น?"
+    , faqQuestion3 = "ถ้าเว็บไซต์นี้ฟรีและไม่ได้วางโฆษณาหรือขายข้อมูล มันสามารถรักษาตัวเองได้อย่างไร?"
+    , forHelpingMeOutWithPartsOfTheApp = " สำหรับการช่วยฉันในส่วนของแอป"
+    , formatDate = Date.format "MMMM ddd"
+    , frequentQuestions = "คำถามที่พบบ่อย"
+    , futureEvents = "กิจกรรมในอนาคต"
+    , goToHomepage = "ไปที่หน้าหลัก"
+    , group1 = "คุณยังไม่ได้สมัครสมาชิกกับกลุ่มใด ๆ คุณสามารถทำได้โดยการกด \""
+    , groupDescription = "คำอธิบายกลุ่ม"
+    , groupName = "ชื่อกลุ่ม"
+    , groupNotFound = "ไม่พบกลุ่ม"
+    , hideU_00A0Attendees = "(ซ่อน\u{00A0}ผู้เข้าร่วม)"
+    , hoursLong = " ชั่วโมง."
+    , howManyHoursLongIsIt = "มันยาวกี่ชั่วโมง?"
+    , howManyPeopleCanJoinLeaveThisEmptyIfThereSNoLimit = "มีกี่คนที่สามารถเข้าร่วม (ปล่อยว่างถ้าไม่มีขีดจำกัด)"
+    , ifYouDontSeeTheEmailCheckYourSpamFolder = "ถ้าคุณไม่เห็นอีเมล ตรวจสอบโฟลเดอร์สแปมของคุณ"
+    , imageEditor = "ตัวแก้ไขรูปภาพ"
+    , info = "ข้อมูล"
+    , inPersonEvent = "กิจกรรมที่พบกันด้วยตัวเอง 🤝"
```

Comment:
probably mistranslated

Reply:
@CharlonTank I guess I should wait with approving the merge until this is fixed?

Reply:
Oh I didn't see that one, yes, we should wait until I get verified by some Thai speaking people :)

Reply:
_Key: `inPersonEvent`_

**Original**: in-person event 🤝

**Old Translation**: กิจกรรมที่พบกันด้วยตัวเอง 🤝

_Dtinth Review: Probably mistranslated_

**New Translation**: กิจกรรมที่เจอกันด้วยตัวเอง 🤝

**Why**: The new translation is a more natural way to express "in-person event" in Thai.

Reply:
```suggestion
    , inPersonEvent = "กิจกรรมที่เจอกันด้วยตัวเอง 🤝"
```